load/UpdateDBPAccessTable.py

```diff
@@ -98,10 +98,10 @@ def _isPublicDomain(self, record):
 
 if (__name__ == '__main__'):
 	config = Config()
-	lptsReader = LPTSExtractReader(config.filename_lpts_xml)
+	languageReader = LanguageReaderCreator().create(config)
```

Comment:
We should import the LanguageReaderCreator in the main to execute the UpdateDBPAccessTable.py file locally. e.g.

```python
from LanguageReaderCreator import LanguageReaderCreator
```